cpp/ql/src/semmle/code/cpp/ir/internal/ASTValueNumbering.qll

```diff
@@ -57,7 +57,7 @@ class GVN extends TValueNumber {
 
   final Location getLocation() {
     result =
-      rank[1](Location l |
+      min(Location l |
```

Comment:
The same note applies re: `UnknownLocation` and implicit `this` qualifiers.

Reply:
Fixed.